File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = s.toLowerCase().split(" ")
    
    for(let i=0;i<answer.length;i++){ 
        if(/^[A-Za-z]$/.test(answer[i][0])){
            answer[i] = answer[i][0].toUpperCase() + answer[i].slice(1) }
    }
    return answer.join(" ");
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.kt
class Solution {
    fun solution(s: String): String {
        var answer = ""
        var str = s.lowercase().split(" ")
        
        //map 사용
        var strArr = str.map{
            if(it.isNotEmpty()) it[0].uppercaseChar() + it.substring(1)
            else it
            
        }
        answer = strArr.joinToString(" ")
        return answer
    }
}